docs/installation/debian.md

```diff
@@ -49,17 +49,21 @@ from the new repository:
          $ apt-get purge lxc-docker*
          $ apt-get purge docker.io*
 
- 4. Add the new `gpg` key.
+ 4. Ensure that APT can work with `https` method and that CA certificates are installed.
+
+         $ apt-get install apt-transport-https ca-certificates
```

Comment:
was wondering if we should add an `apt-get update` before this, but we're also doing that in step 10., so it's probably fine like this